packages/suite-base/src/testing/builders/PlayerStateBuilder.ts

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+import {
+  PlayerPresence,
+  PlayerState,
+  PlayerStateActiveData,
+} from "@lichtblick/suite-base/players/types";
+import BasicBuilder from "@lichtblick/suite-base/testing/builders/BasicBuilder";
+import RosTimeBuilder from "@lichtblick/suite-base/testing/builders/RosTimeBuilder";
+import { defaults } from "@lichtblick/suite-base/testing/builders/utilities";
+import { OptionalMessageDefinition, RosDatatypes } from "@lichtblick/suite-base/types/RosDatatypes";
+
+class PlayerStateBuilder {
```

Comment:
@laisspportugal I've just noticed that **PlayerBuilder** already implements the **PlayerState** data.
So, I'd say to remove this file e reuse from **PlayerBuilder**

Reply:
... and my last comments in this PR became irrelevant

Reply:
Fixed